ttnn/cpp/ttnn/operations/transformer/concatenate_heads/concatenate_heads.cpp

```diff
@@ -5,11 +5,11 @@
 #include "concatenate_heads.hpp"
 
 
-#include "ttnn/deprecated/tt_dnn/op_library/nlp_tms/nlp_tms.hpp"
+#include "ttnn/cpp/ttnn/operations/experimental/transformer/nlp_concat_heads/device/nlp_concat_heads_device_operation.hpp"
 
 namespace ttnn::operations::transformer {
 
-struct ConcatenateHeads : public tt::tt_metal::NlpConcatHeads {
+struct ConcatenateHeads : public ttnn::operations::experimental::transformer::NLPConcatHeadsDeviceOperation {
```

Comment:
We probably need to move ConcatenateHeads to experimental too.
@TT-BrianLiu @sjameelTT  Wdyt

Reply:
All the split/concatenate heads seem pretty non-generic and don't have a corresponding torch api. I feel that they're all better suited for experimental.